examples/todomvc/webpack.config.js

```diff
@@ -2,18 +2,34 @@ const path = require('path')
 
 module.exports = {
   devServer: {
-    contentBase: [
+    static: [
       path.resolve(__dirname),
       path.resolve(__dirname, '../../packages/')
     ]
   },
   externals: {
     'conduit-rxjs': 'Conduit',
     'conduit-rxjs-react': 'ConduitReact',
-    'react': 'React',
+    react: 'React',
     'react-dom': 'ReactDOM',
-    'router': 'Router',
-    'rxjs': 'rxjs',
+    router: 'Router',
+    rxjs: 'rxjs',
     'rxjs/operators': 'rxjs.operators'
```

Comment:
As of [RxJS v7.2](https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md#720-2021-07-05) (2021-07-05):

> Operators are all exported at the top level, from "rxjs". From here on out, we encourage top-level imports with RxJS. Importing from rxjs/operators will be deprecated soon.

So, if we upgrade the examples to the latest RxJS, then we can clean up this "externals" line — here and in all the examples. We could also force the minimum RxJS version to be v7.2, if we wanted. It would primarily affect `conduit-rxjs-react` since it uses operators, while `conduit-rxjs` does not. But if we force a minimum version on one, we may as well do the same for the other.

Reply:
👍 Since we're forcing React updates we might as well also force RxJS updates.

Reply:
Would it be appropriate to replace the references to rxjs/operators in the CHANGELOG.md files too?

Reply:
The README files and examples should be updated. But operator imports mentioned in prior releases in the CHANGELOG files should be kept as-is. It should reflect the code at that point in time. This change should be mentioned in the CHANGELOG for this upcoming version.

Reply:
Since all operators are now a top-level import, I'd have one import line, rather than two.

```js
// Update this:
import { interval } from 'rxjs'
import { map, startWith, switchMap } from 'rxjs'

// To this (organizing imports alphabetically):
import { interval, map, startWith, switchMap } from 'rxjs'
```